chapters/07_chapter_6.tex

```diff
@@ -0,0 +1,390 @@
+\chapter{Introduction to micro-architectural attacks}
+
+
+
+Clémentine Maurice, CNRS, IRISA\\
+April 30, 2019 — Ben Gurion University, Israel
+
+\label{c7_cacheattacks}
+
+\section{Background \& Primitives}\label{arbel:sec}
```

Comment:
This should be of the following format: {sec:arbel}. Same for subsection: {subsec:AAA}

Reply:
Done